test/e2e/common/node/sysctl.go

```diff
@@ -105,7 +111,13 @@ var _ = SIGDescribe("Sysctls [LinuxOnly] [NodeFeature:Sysctls]", func() {
 		gomega.Expect(log).To(gomega.ContainSubstring("kernel.shm_rmid_forced = 1"))
 	})
 
-	ginkgo.It("should support unsafe sysctls which are actually whitelisted", func() {
+	/*
+	  Release: v1.21
+	  Testname: Sysctl, allow whitelisted unsafe sysctls
```

Comment:
nit: s/whitelisted/allow-listed/ or s/whitelisted/allowed/ in these test names
 
 The flag seems to be `allowed`, no reason to use `whitelisted` instead of `allowed` in the test names.
https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#enabling-unsafe-sysctls
https://github.com/kubernetes/community/pull/5341

Reply:
I agree with this.  I won't block on this though, under the assumption that we can correct this post-test-freeze as a comment-only change (though it will result in changing the contents of conformance.yaml)

Reply:
fixed !